lib/helpers.js

```diff
@@ -232,11 +235,36 @@ function writeSourceMaps(file, destPath, callback) {
   }
 
   // Append source map comment
-  file.contents = appendBuffer(file.contents, comment);
+  file.contents = appendBuffer(file.contents, new Buffer(comment));
```

Comment:
```suggestion
  file.contents = appendBuffer(file.contents, Buffer.from(comment));
```

Should this be Buffer.from?